src/common/components/layouts/pages/inspect/validationErrorMessages_en.json

```diff
@@ -172,6 +172,83 @@
       "26": {
         "SUMMARY": "Is this a table?",
         "DESCRIPTION": "This text looks like a table but it's tagged as a list. If you intended this to be a table it's better to format it as such."
+      },
+      "27": {
+        "SUMMARY": "Are these different paragraphs?",
+        "DESCRIPTION": "This text is tagged as a single paragraph but appears to be several. Always use paragraph breaks (<Enter>), don't use line breaks (<Shift><Enter>) to delimit paragraphs. In Word you can see paragraph breaks by going to the File - Options - Display menu and selecting Paragraph marks there."
```

Comment:
`File - Options - Display` Maybe better replace `-` to `->` as in other descriptions